client/ayon_core/hosts/houdini/plugins/publish/collect_redshift_rop.py

```diff
@@ -60,13 +60,19 @@ def process(self, instance):
             instance.data["ifdFile"] = beauty_export_product
             instance.data["exportFiles"] = list(export_products)
 
-        # Default beauty AOV
+        full_exr_mode = (rop.evalParm("RS_outputMultilayerMode") == "2")
+        if full_exr_mode:
+            # Ignore beauty suffix if full mode is enabled
+            # As this is what the rop does. 
+            beauty_suffix = ""
+
+        # Default beauty/main layer AOV
         beauty_product = self.get_render_product_name(
             prefix=default_prefix, suffix=beauty_suffix
         )
         render_products = [beauty_product]
         files_by_aov = {
-            "_": self.generate_expected_files(instance,
+            beauty_suffix: self.generate_expected_files(instance,
```

Comment:
```suggestion
            beauty_suffix: self.generate_expected_files(instance, beauty_product)}
```
More likely cosmetic(without hound we didn't notice this) maybe we can just move this argument from line 76 back to line 75

Reply:
I've just tried it and found out the line will be 83 characters long.
where max should be 79.

https://github.com/ynput/ayon-core/blob/c3860e131136a92ee727c79f1ad222103719dfb9/setup.cfg#L4